container/karaf/src/main/filtered-resources/etc/system.properties

```diff
@@ -177,10 +177,13 @@ karaf.handle.sigterm = false
 # @see https://issues.opennms.org/browse/NMS-9416
 # @see https://github.com/apache/mina-sshd/blob/master/sshd-core/src/main/java/org/apache/sshd/common/util/security/SecurityUtils.java
 #
+# The WMI enhancements of NMS-12788 require Bouncy Castle in the system classpath, so the change is now configured by default.
+# @see https://issues.opennms.org/browse/NMS-12788
+
 # Original:
 #org.apache.sshd.security.registrars = \
 #  org.apache.sshd.common.util.security.bouncycastle.BouncyCastleSecurityProviderRegistrar,\
 #  org.apache.sshd.common.util.security.eddsa.EdDSASecurityProviderRegistrar
 #
 # Edited:
-#org.apache.sshd.security.registrars = org.apache.sshd.common.util.security.eddsa.EdDSASecurityProviderRegistrar
+org.apache.sshd.security.registrars = org.apache.sshd.common.util.security.eddsa.EdDSASecurityProviderRegistrar
```

Comment:
Still need this system property ?

Reply:
Yes - the SSH bundle won't start in the main OpenNMS karaf container without it.